### ISSUE TITLE
[tests] SDPACausal and SDPAMasked set mask value to `null`

### ISSUE THREAD

**Issue by periannath** [Samsung/TICO#351]
# What

SDPACausal and SDPAMasked set mask value to `null`. This value should be `-inf`.

## SDPACausal

<img width="500" height="790" alt="Image" src="https://github.com/user-attachments/assets/c2512021-d374-4935-9124-d2eac96728ef" />

## SDPAMasked

<img width="150" height="290" alt="Image" src="https://github.com/user-attachments/assets/ea9be079-8ba9-4b5d-b5aa-3c7e3805dbc5" />

```python
# Efficient implementation equivalent to the following:
def scaled_dot_product_attention(query, key, value, attn_mask=None, dropout_p=0.0,
        is_causal=False, scale=None, enable_gqa=False) -> torch.Tensor:
    L, S = query.size(-2), key.size(-2)
    scale_factor = 1 / math.sqrt(query.size(-1)) if scale is None else scale
    attn_bias = torch.zeros(L, S, dtype=query.dtype, device=query.device)
    if is_causal:
        assert attn_mask is None
        temp_mask = torch.ones(L, S, dtype=torch.bool).tril(diagonal=0)
        attn_bias.masked_fill_(temp_mask.logical_not(), float("-inf"))
        attn_bias.to(query.dtype)
```
From https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html

As above code shows, this value should be set to `-inf` instead of `null`.

**Comment by mhs4670go** [comment]
Umm.. in my local PC, the values are -Infinity. Could you cross-check with `circledump` or something? It could be a netron problem.
